### DISCUSSION TITLE
How to select/delete files on left/right folder but not present on other folder?

### DISCUSSION THREAD

**Discussion by teasz** [WinMerge/winmerge#1285] [Q&A]
I wanted to delete files that are present on LEFT but not present on RIGHT folder without deleteing the entire left folder?

**Comment by sdottaka** [comment]
First, check "Include unique subfolders contents" in the "Compare/Folder" category of the Options dialog if it is unchecked.

![image](https://user-images.githubusercontent.com/98126/161971071-6c608445-1e33-45bf-8dcc-d235c6899011.png)

Then, compare the folders, select the files as shown below, right-click and select Delete => Left menu item.

![image](https://user-images.githubusercontent.com/98126/161971518-1a141bc6-fa67-4509-b3a2-151ef6d34825.png)

**Reply by jmanos** [reply]
Using this method, it is helpful to sort by the "Comparison result" column before deleting if you have non-adjacent files that need to be deleted.
